### PR TITLE
(BIDS-1681) refactor the currency handling of the layout.html

### PR DESCRIPTION
**Pull request by MauserBitfly**

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -182,9 +106,8 @@
                   <div class="info-item-body">
                     <a id="banner-eth-price-data">
                       <span id="currentCurrencySymbol" class="currency-symbol">{{ .Rates.CurrentSymbol }}</span>
-                      <span id="initialPrice">{{ .Rates.CurrentPriceFormatted }}</span>
-                      <span id="currentKFormattedPrice" class="k-formatted-price"></span>
-                      <span id="currentCurencyPrice" class="price"></span>
+                      <span id="currentKFormattedPrice" class="k-formatted-price">{{ .Rates.CurrentPriceKFormatted }}</span>
+                      <span id="currentCurencyPrice" class="price">{{ .Rates.CurrentPriceFormatted }}</span>
```
Comment (LuccaBitfly) [inline templates/layout.html:110]: typo in id, pls fix

**Review by LuccaBitfly — CHANGES_REQUESTED** [review]

File: utils/utils.go [hunk 2]
```diff
@@ -130,6 +130,8 @@ func GetTemplateFuncs() template.FuncMap {
 		"formatAddressAsLink":                     FormatAddressAsLink,
 		"formatBuilder":                           FormatBuilder,
 		"formatDifficulty":                        FormatDifficulty,
+		"getAvailableCurrencies":                  price.GetAvailableCurrencies,
```
Comment (LuccaBitfly) [inline utils/utils.go:133]: Any reason why the available currencies are passed as a template function instead of an extra field in PageData?

**Comment by LuccaBitfly** [comment]
otherwise lgtm

File: utils/utils.go (hunk 2 above)
Comment (MauserBitfly) [reply utils/utils.go]: I had one ... but after thinking about - you are right ... put it in the PageData now

**Review by LuccaBitfly — APPROVED** [review]
lgtm!

**Pull request merged**